common/changes/@itwin/core-backend/nick-add-querycurrentprecedingv2_2022-12-01-19-37.json

```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "@itwin/core-backend",
-      "comment": "add queryCurrentOrPrecedingV2Checkpoint as an optional function to backendhubaccess interface",
+      "comment": "add allowPreceding as an optional boolean to CheckpointProps interface",
```

Comment:
Also modified rush chanhge from a previous PR. After requested changes, the old message: "add queryCurrentOrPrecedingV2Checkpoint as an optional function to backendhubaccess interface" was not true.